install/kubernetes/helm/istio/values.yaml

```diff
@@ -172,6 +172,16 @@ global:
     # Configure the access log for sidecar to JSON or TEXT.
     accessLogEncoding: TEXT
 
+    # Configure envoy gRPC access log service.
+    envoyAccesslogService:
+      enable: false
+      config:
+        address: ""
```

Comment:
instead of address, should we use `host` and `port` a la `envoyMetricsService` ?

Reply:
When the API PR gets done, I will update this configuration according to it. Now let's not change it.

Reply:
done